### PR TITLE
fix: nitpicks/add some details for theme tokens

### PR DESCRIPTION
**Pull request by nhidtran**

### CODE REVIEW

File: docs/foundations/shadow.mdx [hunk 1]
```diff
@@ -4,6 +4,8 @@ title: Shadow
 description: Shadow tokens are predefined box-shadow values.
 ---
```
Comment (BrianAA) [inline docs/foundations/shadow.mdx:6]: add a `<br/>` for spacing

File: docs/foundations/color.mdx [hunk 2]
```diff
@@ -4,6 +4,8 @@ title: Color
 description: All color tokens define the color in the current context. There are two contexts `light` and `dark` context. Toggle the site's context to see each token color value.
 ---
 
+When using color tokens in code, you can access them using `theme.colors.[tokenName]`
```
Comment (BrianAA) [inline docs/foundations/color.mdx]: Move this below to the Theme and Semantic section Line 22

File: docs/foundations/size.mdx [hunk 3]
```diff
@@ -6,6 +6,7 @@ description:
 
 Size tokens are relative to the [baseSize](/foundations/base).
 Use size tokens to specify a fixed width or height when designing a component or layout.
+When using size tokens in code, you can access them using `theme.size.[tokenName]`
```
Comment (BrianAA) [inline docs/foundations/size.mdx]: add a `<BR/>` for spacing

**Review by BrianAA — CHANGES_REQUESTED** [review]
Just small spacing refinements and placement

**Review by artmsilva — APPROVED** [review]

**Comment by artmsilva** [comment]
@nhidtran Great job!! Thank you for contributing to our docs.

**Pull request merged**